js/okx.js

```diff
@@ -2064,6 +2064,10 @@ module.exports = class okx extends Exchange {
             // 'slTriggerPxType': 'last', // Conditional default is last, mark or index (conditional orders)
             // 'slOrdPx': 10, // Order price for Stop-Loss orders, if -1 will be executed at market price (conditional orders)
         };
+        const reduceOnly = this.safeValue (params, 'reduceOnly', false);
```

Comment:
@Dan-krm do we really need to do this?

Basically we're doing this:

```
params = {"reduceOnly":true}
request = {}
```
⬇️ 

```
request = {"reduceOnly":true"}
params = {}
```
but in the end we do this:

```
this.extend(request, params)
```
So I don't think it will make any difference

Reply:
I don't think it makes a difference, thanks for explaining, closing this PR now